04_setup_ironic.sh

```diff
@@ -70,14 +70,14 @@ fi
 sudo podman pod create -n ironic-pod 
 
 # Pull the rhcos-downloder image to use from the release, this gets change
```

Comment:
nit: let's change this also

Reply:
Good catch, but lets fix that as a follow-up so we can unblock CI